kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -272,6 +272,19 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  val CREDENTIALS_HIVE_EXTRA_METASTORE_URIS: ConfigEntry[Seq[String]] =
```

Comment:
why do we need this?

Reply:
Kyuubi administrator can set this conf to let kyuubi renew token for extra Hive metastore servers, which are not in hive-site.xml.

Without this, if user specified any other Hive metastore server in JDBC url,  kyuubi engine can not access it after the initial token expires.

Reply:
> Kyuubi administrator can set this conf to let kyuubi renew token for extra Hive metastore servers, which are not in hive-site.xml.

why didn't he modify the hive-site.xml directly?

Reply:
User specified Hive metastore server may not belong to the Hive metastore cluster defined in hive-site.xml.

As this is a patch of `HiveDelegationTokenProvider`, I'll remove it from this PR.